openstates/scrape/base.py

```diff
@@ -126,6 +134,7 @@ def save_object(self, obj):
         filename = "{0}_{1}.json".format(obj._type, obj._id).replace("/", "-")
 
         self.info("save %s %s as %s", obj._type, obj, filename)
```

Comment:
Can we be consistent with string formatting? I don't really care _what_ format, but let's not do c-strings _and_ f-strings in the same file?

Reply:
alright, changed

Reply:
Above this comment, we have two distinct string formatting patterns right next to each other. I think that _generally_ OpenStates leans towards f-strings, but as long as the file is at least internally consistent, that works.